pkg/graph/schema/types/mto_solution.graphql

```diff
@@ -33,3 +35,23 @@ type MTOSolution {
   addedFromSolutionLibrary: Boolean!
   commonSolution: MTOCommonSolution
 }
+
+input MTOSolutionChanges @goModel(model: "map[string]interface{}") {
+  name: String # If sourced from a Common Solution, this will resolve from the common solution table (you can't edit this for common solution!)
+  facilitatedBy: MTOFacilitator
+  status: MTOSolutionStatus
+  riskIndicator: MTORiskIndicator
+  key: MTOCommonSolutionKey
```

Comment:
Can you please take another look at what fields should be exposed in this Changes Struct? Notably, I think that `key` and `type` should not be included here ([figma](https://www.figma.com/design/BcI7pNwfkmwaDD2EGGTsYC/IT-Lead?node-id=4339-89395&node-type=frame&t=uqJnchVYFtXgZTG3-0) )

Reply:
On reviewing this I noticed we were missing the needed_by field as well so I added it to the relevant areas.